utils_cv/detection/dataset.py

```diff
@@ -53,8 +109,8 @@ def __init__(
         batch_size: int = 2,
         transforms: object = get_transform(train=True),
         train_pct: float = 0.5,
-        im_dir: str = "images",
         annotation_dir: str = "annotations",
+        im_dir: str = None,
```

Comment:
shouldn't the default behavior be "images" instead of None?

Reply:
Good point.